src/nl/igorski/lib/audio/MWEngine.java

```diff
@@ -56,22 +56,10 @@
     public static int SAMPLE_RATE = 44100;
     public static int BUFFER_SIZE = 2048;
 
-    /* time signature related */
-
-    public static int TIME_SIG_BEAT_AMOUNT  = 4; // upper numeral in time signature (i.e. the "3" in 3/4)
-    public static int TIME_SIG_BEAT_UNIT    = 4; // lower numeral in time signature (i.e. the "4" in 3/4)
-
     // we CAN multiply the output the volume to decrease it, preventing rapidly distorting audio ( especially on filters )
     private static final float VOLUME_MULTIPLIER = .85f;
     private static float       _volume           = .85f /* assumed default level */ * VOLUME_MULTIPLIER;
 
-    /* we make these statically available to outside classes, these changes according to time signature and tempo */
-
```

Comment:
The values moved to the SequencerController existed as static properties on the MWEngine class, but that made very little sense!